### ISSUE TITLE
Improve documentation

### ISSUE THREAD

**Issue by j-harbin** [casaultb/azmpdata#36]
As discussed in previous discussions, I use issues as a "to-do" list. Again, I am unable to assign this issue to myself.

**Comment by j-harbin** [comment]
Hi All,

I’ve made a list of suggestions for documentation. Feel free to take or leave any of my suggestions below

1. In the help docs for `variable_lookup`, there is a typo. For `search_help` is should say a “logical” rather than “logcial”. It may also be helpful to add an example. For example, 

```r
library(azmp) 
res <- variable_lookup('nitrate') 
res2 <- variable_lookup("nitrate", search_help=TRUE) 
```

It may also be a good idea to have the authors of this documentation.

2. In the help docs for `update_check`, I **think** GitHub is supposed to be capitalized. It also may be a good idea to have Emily’s contact info under the author list as well
3. For the help docs for `read.physical`, it may again be a good idea to have an example of this. It is a bit unclear as a user if the file would be one that I would go find myself, or if it is from one of the built in datasets, or both?
4. This is absolutely me being picky, but when I look at the index page (screenshot below), it may look nice if in all of the descriptions the word “Annual” was capitalized. I see some times it is, and some times it isn’t. I would definitely consider this low priority, but worth mentioning.

![Screen Shot 2021-05-31 at 8 02 46 AM](https://user-images.githubusercontent.com/58750538/120184031-e1364080-c1e6-11eb-8cc0-d25f45e1725b.png)

I hope this helps :)

**Comment by j-harbin** [comment]
Hi All,

I added these updates to my pull request to mmm_dev. It should be noted that:
1.  I did not know the author(s) for variable_lookup, so I did not add that in
2. I no longer see read.physical in the docs. Is this an error on my end?
3. Shown in screen shot below, when we looked on the "index" page, for functions such as df2sf and plot_availability, etc. the title was simply the name of the function, so I changed it to be more of a description, similar to variable_lookup()

![Screen Shot 2021-08-17 at 1 41 25 PM](https://user-images.githubusercontent.com/58750538/129766545-9ca7fb4f-ac9e-437a-9786-83e08df52716.png)
